config.go

```diff
@@ -38,7 +38,9 @@ func parseProperties(reader io.Reader) map[string]string {
 
 		split := strings.Split(line, "=")
 		if len(split) < 2 {
-			log.Printf("WARN: cannot parse config line %s\n", line)
+			if len(line) > 0 {
+				log.Printf("WARN: cannot parse config line %s\n", line)
```

Comment:
:+1:

Reply:
I wonder if most if this code is useful or not, Viper has native support for Properties files.
(It's not something to fix in the scope of THIS PR)